src/App.js

```diff
@@ -1,16 +1,51 @@
 import React, { Component } from 'react';
+
 import Navbar from './containers/Navbar/Navbar';
 import Home from './containers/Home/Home';
+import NewPost from './containers/NewPost/NewPost';
+import Profile from './containers/Profile/Profile';
+import asyncComponent from './hoc/asyncComponent';
+import NoLogin from './components/NoLogin/NoLogin';
+
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
```

Comment:
This is a small thing but when writing code for React Components. The order of the imports should be that `react` should be imported first, then all external packages (`node_modules`), and then any components or util functions.